.github/workflows/CD.yml

```
@@ -61,14 +83,24 @@ jobs:
           username: ${{ secrets.DOCKERHUB_LOGIN_USERNAME }}
           password: ${{ secrets.DOCKERHUB_LOGIN_ACCESSTOKEN }}
 
-      - name: Build and Push Docker Image
+      - name: Build and Push Docker Image (develop -> dev-latest)
+        if: ${{ github.ref_name == 'develop' }}
         uses: docker/build-push-action@v4
```

actionlint (1.7.11)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)